main.go

```diff
@@ -85,9 +87,50 @@ func main() {
 			Usage:   "Approximates the rETH stakers' share of the Smoothing Pool at the current block instead of generating the entire rewards tree. Ignores -i.",
 			Value:   false,
 		},
+		&cli.StringFlag{
+			Name:    "cpuprofile",
+			Aliases: []string{"c"},
+			Usage:   "Path to which to save a pprof cpu profile, e.g. ./treegen.pprof. If unset, profiling is disabled.",
+		},
+		&cli.StringFlag{
+			Name:    "memprofile",
+			Aliases: []string{"m"},
+			Usage:   "Path to which to save a pprof heap profile, e.g. ./treegen.pprof. If unset, profiling is disabled.",
+		},
 	}
 
 	app.Action = func(c *cli.Context) error {
+		cpuprofile := c.String("cpuprofile")
+		if cpuprofile != "" {
+			f, err := os.Create(cpuprofile)
+			if err != nil {
+				fmt.Printf("%sError generating tree: %s%s\n", colorRed, err.Error(), colorReset)
```

Comment:
Should this say "Error saving CPU profile" to have parity with the heap profile error below?